src/main/java/uk/gov/hmcts/reform/et/syaapi/service/NotificationService.java

```diff
@@ -390,114 +377,87 @@ public void sendAcknowledgementEmailToTribunal(CaseData caseData,
     /**
      * Format details of claimant request and retrieve case data, then send email to confirmation to tribunal.
      *
-     * @param caseData        existing case details
-     * @param claimant        claimant's full name
-     * @param caseNumber      ethos case reference
-     * @param respondentNames concatenated respondent names
-     * @param hearingDate     date of the nearest hearing
-     * @param caseId          16 digit case id
+     * @param details core details of the email
      * @param applicationType type of application
      * @param isRespondingToRequestOrOrder indicates whether the reply is to a tribunal order or not
      */
-    public void sendResponseEmailToTribunal(
-        CaseData caseData,
-        String claimant,
-        String caseNumber,
-        String respondentNames,
-        String hearingDate,
-        String caseId,
-        String applicationType,
-        boolean isRespondingToRequestOrOrder
-    ) {
-        String subjectLine = String.format("%s %s", caseNumber, applicationType);
+    void sendResponseEmailToTribunal(CoreEmailDetails details, String applicationType,
+                                     boolean isRespondingToRequestOrOrder) {
+        String subjectLine = String.format(TWO_STRINGS_PATTERN, details.caseNumber, applicationType);
```

Comment:
Not sure this is an improvement, TWO_STRINGS_PATTERN isn't clear how it'll look. This might an over refactor if its a stand in for "%s %s"

Reply:
Haha yeah I didn't love it, but sonarlint was complaining. I'll just concatenate the two.

Reply:
Try `String.join(" ", caseNumber, applicationType);`